homeassistant/components/netatmo/strings.json

```diff
@@ -121,18 +121,50 @@
       "description": "Unregisters the webhook from the Netatmo backend."
     },
     "set_preset_mode_with_end_datetime": {
-      "name": "Set preset mode with end datetime",
+      "name": "Set preset mode with end date & time",
       "description": "Sets the preset mode for a Netatmo climate device. The preset mode must match a preset mode configured at Netatmo.",
       "fields": {
         "preset_mode": {
           "name": "Preset mode",
           "description": "Climate preset mode such as Schedule, Away or Frost Guard."
         },
         "end_datetime": {
-          "name": "End datetime",
-          "description": "Datetime for until when the preset will be active."
+          "name": "End date & time",
+          "description": "Date & time the preset will be active until."
         }
       }
+    },
+    "set_temperature_with_end_datetime": {
+      "name": "Set temperature with end date & time",
+      "description": "Sets the target temperature for a Netatmo climate device with an end date & time.",
+      "fields": {
+        "target_temperature": {
+          "name": "Target temperature",
+          "description": "The target temperature for the device."
+        },
+        "end_datetime": {
+          "name": "End date & time",
```

Comment:
Please use "links" when the phrases matches up
Example `[%key:component::netatmo::services::xyz::xyz%]`

Reply:
Think that should have sorted it.

Thanks for your patience with this PR. The quick responses have certainly been appreciated.